doc/installation_and_administration_guide/install_from_sources.md

```diff
@@ -62,8 +62,6 @@ Apache Flume can be easily installed by downloading its latests version from [fl
     $ mv apache-flume-1.4.0-bin APACHE_FLUME_HOME
     $ mv APACHE_FLUME_HOME/lib/httpclient-4.2.1.jar APACHE_FLUME_HOME/lib/httpclient-4.2.1.jar.old
     $ mv APACHE_FLUME_HOME/lib/httpcore-4.2.1.jar APACHE_FLUME_HOME/lib/httpcore-4.2.1.jar.old
-    $ wget http://repo1.maven.org/maven2/org/apache/thrift/libthrift/0.9.1/libthrift-0.9.1.jar -P APACHE_FLUME_HOME/lib
-    $ mv APACHE_FLUME_HOME/lib/libthrift-0.7.0.jar APACHE_FLUME_HOME/lib/libthrift-0.7.0.jar.old
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/4374a6b983ae9dce1296548d9e78ec16a595cf9b